policyengine_us/parameters/gov/states/nj/tax/exemptions/regular/amount.yaml

```diff
@@ -0,0 +1,19 @@
+description: NJ provides a regular exemption of $1k for individuals and $2k for couples.
+metadata:
+  unit: currency-USD
+  label: NJ Regular Exemption
+  reference:
+    - title: Instructions for Form NJ-1040
```

Comment:
```suggestion
    - title: Instructions for Form NJ-1040 (line 6)
```

Reply:
Got it, will follow this format! Thanks for the clarification